src/tls.c

```diff
@@ -66,6 +66,16 @@
 #endif
 
 
+#ifndef WOLFSSL_HAVE_MIN
+#define WOLFSSL_HAVE_MIN
+
+    static INLINE word32 min(word32 a, word32 b)
+    {
+        return a > b ? b : a;
+    }
+
+#endif /* WOLFSSL_HAVE_MIN */
+
```

Comment:
This isn't required here because we have the misc.c logic at the top. This section with WOLFSSL_HAVE_MIN can be remove.